### ISSUE TITLE
@TracedAccessors generates methods for "class" feature that does not exist

### ISSUE THREAD

**Issue by spoenemann** [eclipse/xtext-core#349]
Using one of the `_class` extension methods leads to

```
java.lang.NullPointerException
	at org.eclipse.xtext.generator.trace.LocationData.<init>(LocationData.java:26)
	at org.eclipse.xtext.generator.trace.node.TracingSugar.location(TracingSugar.java:89)
	at io.typefox.traceexample.eclipse.generator.MyDslGenerator$MyDslTraceExtensions._class(MyDslGenerator.java:78)
	at io.typefox.traceexample.eclipse.generator.MyDslGenerator$3.appendTo(MyDslGenerator.java:727)
	at org.eclipse.xtend2.lib.StringConcatenationClient.appendTo(StringConcatenationClient.java:50)
	at org.eclipse.xtext.generator.trace.node.TemplateNode.<init>(TemplateNode.java:39)
	at org.eclipse.xtext.generator.trace.node.GeneratorNodeExtensions.appendTemplate(GeneratorNodeExtensions.java:142)
	at io.typefox.traceexample.eclipse.generator.MyDslGenerator.generateHeader(MyDslGenerator.java:464)
	at io.typefox.traceexample.eclipse.generator.MyDslGenerator$1.appendTo(MyDslGenerator.java:411)
	at org.eclipse.xtend2.lib.StringConcatenationClient.appendTo(StringConcatenationClient.java:50)
	at org.eclipse.xtext.generator.trace.node.TemplateNode.<init>(TemplateNode.java:39)
	at org.eclipse.xtext.generator.trace.node.GeneratorNodeExtensions.appendTemplate(GeneratorNodeExtensions.java:142)
	at org.eclipse.xtext.generator.trace.node.TracingSugar.trace(TracingSugar.java:67)
	at org.eclipse.xtext.generator.trace.node.TracingSugar.generateTracedFile(TracingSugar.java:44)
	at io.typefox.traceexample.eclipse.generator.MyDslGenerator.doGenerate(MyDslGenerator.java:421)
```

The generated method that causes the exception is

```java
  public IGeneratorNode _class(final ClassDeclaration target, final Function<Class<?>, String> stringProvider) {
      EStructuralFeature feature = target.eClass().getEStructuralFeature("class");
      ILocationData location = this.location(target, feature, -1);
      CompositeGeneratorNode trace = this.trace(location);
      this.append(trace, stringProvider.apply(target.getClass()));
      return trace;
  }
```

Such a structural feature `"class"` does not exist.

**Comment by cdietrich** [comment]
maybe we should skip stuff from java.lang.Object

**Comment by cdietrich** [comment]
Fixed in 2.13
